src/authenticatedFetch/unauthenticated/UnauthenticatedFetcher.ts

```diff
@@ -67,6 +68,9 @@ export default class DpopAuthenticatedFetcher implements IAuthenticatedFetcher {
     url: RequestInfo,
     requestInit?: RequestInit
   ): Promise<Response> {
+    if (requestInit?.headers) {
+      console.log("Received headers");
```

Comment:
To be removed

Reply:
Also, a linter rule likely should have caught this.

Reply:
Resolved in d084878001cd35bdca8cc97b48d46423fad32c1a. And agreed, enforcing the same linter rules on all our projects is a task that's being completed right now.